src/styles/blocks/catalog.scss

```diff
@@ -17,6 +17,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color 300ms, transform 300ms;
+
```

Comment:
you can use "transition: all 0.3s" for all properties

Reply:
Thanks! I did not know that

Reply:
actually, that is considered to be a bad practice to use `transition: all`
We usually recommend creating a separate variable for the `transition-duration` and using a specific rule.
`
transition-duration: $transition-duration;
transition-property: color, transform;
`